UnityProject/Assets/Scripts/Player/PlayerScript.cs

```diff
@@ -345,7 +345,7 @@ public void SyncPlayerName(string oldValue, string value)
 	/// <summary>
 	/// True if this player is a ghost, meaning they exist in the ghost layer
 	/// </summary>
-	public bool IsGhost => PlayerUtils.IsGhost(gameObject);
+	public bool IsGhost => PlayerState == PlayerStates.Ghost;
```

Comment:
This might break Ai and blob

Reply:
are they checked as is ghost?

I can add the Ai  and  blob if you want

Reply:
The IsGhost is used in some places which the AI and blob need to be checked for as well, i never got around to seeing which stuff really needed it so they are just ghosts atm.

Reply:
ok will add them